src/utils.rs

```diff
@@ -80,7 +80,39 @@ pub fn ident_at(root: &SyntaxNode, offset: usize) -> Option<CursorInfo> {
         }
     }?;
     let parent = ident.node().parent();
-    if let Some(attr) = parent.clone().and_then(Key::cast) {
+    if parent.clone().map_or(false, |x| x.kind() == rnix::SyntaxKind::NODE_INHERIT) {
+        if let Some(node) = parent.and_then(Inherit::cast) {
```

Comment:
```rust
    if parent.clone().map_or(false, |x| x.kind() == rnix::SyntaxKind::NODE_INHERIT) {
        if let Some(node) = parent.and_then(Inherit::cast) {
```

can be simplified into

```rust
if let Some(node) = parent.clone().and_then(Inherit::cast) {
```

the first line is redundant since Inherit::cast only succeeds if the node type is NODE_INHERIT